native/cocos/renderer/gfx-wgpu/WGPUDevice.h

```diff
@@ -28,37 +28,35 @@
 #if (CC_PLATFORM == CC_PLATFORM_WINDOWS)
     #include <windows.h>
 #endif
-
-#include <emscripten/bind.h>
-#include <emscripten/val.h>
-#include "WGPUDef.h"
+#ifdef CC_WGPU_WASM
+    #include "WGPUDef.h"
```

Comment:
Why need to add the `ifdef CC_WGPU_WASM` logic in WebGPU gfx backend?

Reply:
for emscripten wasm only, as you mentioned before we can run wgpu on native by dawn, so there is also CC_WGPU_DAWN, and CC_WGPU_RS

Reply:
That's cool. I got it now.